rare/models/game.py

```diff
@@ -0,0 +1,342 @@
+        otherwise it reports the already known version of the Game for Windows.
+
+        @return str The current version from EGS
+        """
+        if self.igame is not None:
+            return self.game.asset_infos[self.igame.platform].build_version
+        else:
+            return self.game.app_version()
+
+    @property
+    def has_update(self) -> bool:
+        """!
+        @brief Property to report if an InstalledGame has updates available
+
+        Games have to be installed and have assets available to have
+        updates
+
+        @return bool If there is an update available
+        """
+        if self.igame is not None and self.core.lgd.assets is not None:
+            try:
+                if self.remote_version != self.igame.version:
+                    return True
+            except ValueError:
+                logger.error(f"Asset error for {self.game.app_title}")
+                return False
+        return False
+
+    @property
+    def is_installed(self) -> bool:
+        """!
+        @brief Property to report if a game is installed
+
+        This returns True if InstalledGame data have been loaded for the game
+        or if the game is a game without assets, for example an Origin game.
+
+        @return bool If the game should be considered installed
+        """
+        return (self.igame is not None) or self.is_non_asset
+
+    def set_installed(self, installed: bool) -> None:
+        """!
+        @brief Sets the installation status of a game
+
+        If this is set to True the InstalledGame data is fetched
+        for the game, if set to False the igame attribute is cleared.
+
+        @param installed The installation status of the game
+        @return None
+        """
+        if installed:
+            self.igame = self.core.get_installed_game(self.app_name)
+        else:
+            self.igame = None
+        self.set_pixmap()
+
+    @property
+    def can_run_offline(self) -> bool:
+        """!
+        @brief Property to report if a game can run offline
+
+        Checks if the game can run without connectin the internet.
+        It's a simple wrapper around legendary provided information,
+        with handling of not installed games.
+
+        @return bool If the games can run without network
+        """
+        return self.igame.can_run_offline if self.igame is not None else False
+
+    @property
+    def is_foreign(self) -> bool:
+        """!
+        @brief Property to report if a game doesn't belong to the current account
+
+        Checks if a game belongs to the currently logged in account. Games that require
+        a network connection or remote authentication will fail to run from another account
+        despite being installed. On the other hand, games that do not require network,
+        can be executed, facilitating a rudimentary game sharing option on the same computer.
+
+        @return bool If the game belongs to another count or not
```

Comment:
Shouldn't this be "to another account" or what is meant with count here?

Reply:
Yep, will fix